benchmark-ph1ex1/2D-model/2Dgeo.py

```diff
@@ -0,0 +1,21 @@
+'''
```

Comment:
Best practice is for everything in this file to be couched in 
`if __name__ == '__main__':`